frontend/src/components/Hero/Hero.jsx

```diff
@@ -1,65 +1,103 @@
-import React from "react";
-import PropTypes from "prop-types";
-import Button from "@material-ui/core/Button";
-import Grid from "@material-ui/core/Grid";
-import { makeStyles } from "@material-ui/core/styles";
-import Container from "@material-ui/core/Container";
-import { Link } from "react-router-dom";
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import { Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import { Link } from 'react-router-dom';
+import { Box } from '@material-ui/core';
 
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles((theme) => ({
   wrapper: {
-    textAlign: "left",
-    display: "flex",
-    alignItems: "center",
-    width: "100%",
-    height: "80vh",
-    backgroundSize: "cover",
-    backgroundRepeat: "no-repeat"
+    textAlign: 'left',
+    display: 'flex',
+    alignItems: 'center',
+    width: '100%',
+    minHeight: '500px',
+    height: '80vh',
+    backgroundSize: 'cover',
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'center',
+    position: 'relative',
   },
   overlay: {
-    backgroundColor: "white",
-    opacity: "0.7",
-    position: "absolute",
-    top: "0",
-    left: "0",
-    width: "100%",
-    height: "100%"
+    backgroundColor: 'white',
+    opacity: '0.90',
+    position: 'absolute',
+    top: '0',
+    left: '0',
+    width: '100%',
+    height: '100%',
   },
   content: {
-    zIndex: "10"
+    zIndex: '2',
+  },
+  heroTitleContainer: {
+    width: '700px',
```

Comment:
How does this work when the screen is mobile size?

Reply:
I didnt make this component, but I believe we could sort it out with a media query on the same element